src/content/guides/how-to-use-a-wallet/index.md

```diff
@@ -22,7 +22,7 @@ Your address (which is sometimes called a "public key") is like your home addres
 
 You need to provide whoever wants to send you money with your public address. Many wallet apps let you copy your address or show a QR code to scan for easier usage. Avoid typing any Ethereum address manually. This can easily lead to clerical errors and lost funds.
 
-Different apps may vary or use different language, but they should take you through a similar process if you are trying to transfer funds.
+Different apps may vary or use different languages, but they should take you through a similar process if you are trying to transfer funds.
```

Comment:
Language here refers to the terminology/phrasing they use, not the communication system they use (e.g. English, French, etc). 

cc: @konopkja maybe this is confusing?

```suggestion
Different apps may vary or use different language, but they should take you through a similar process if you are trying to transfer funds.
```

Reply:
agree, a clearer "wording" should be used